install.sh

```diff
@@ -531,15 +526,9 @@ if ($INSTALL_NAV); then
 	# Thruster driver
 	sudo apt-get install -qq ros-$ROS_VERSION-roboteq-driver
 
-	instlog "Cloning Navigator Git repositories that need to be built"
-
-	# Software to interface with MIL hardware
-	ros_git_get https://github.com/uf-mil-archive/hardware-common
-
-	# Required steps to build and install lqRRT
-	ros_git_get https://github.com/jnez71/lqRRT.git
-	sudo python $CATKIN_DIR/src/lqRRT/setup.py build
-	sudo python $CATKIN_DIR/src/lqRRT/setup.py install
+	instlog "Performing setup tasks for lqRRT"
+	sudo python $CATKIN_DIR/src/Navigator/gnc/lqRRT/setup.py build
+	sudo python $CATKIN_DIR/src/Navigator/gnc/lqRRT/setup.py install
 
 	# Pull large project files from Git-LFS
 	instlog "Pulling large files for Navigator"
```

Comment:
We could do a separate repo for models and other large files
That way it doesn't take time or resources to clone any individual repo, but you can still get the sim to work without fiddling with git lfs